### PR TITLE
Use Array.find instead of Array.prototype.find

### PR DESCRIPTION
**Pull request by elhesuu**
# Problem
Build is broken on IE due to lack of support (and polyfilling) of Array.prototype.find 

# Solution
Use `Array.find()` instead.

Same issue is experienced with `Array.prototype.includes`, and similar workaround used.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [n/a] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a] Have new automated tests been implemented?
- [n/a] Have new manual tests been written down?
- [] Have tests passed locally?
- [n/a] Have any new strings been translated?

### CODE REVIEW

File: src/components/Camera/index.js [hunk 1]
```diff
@@ -196,7 +196,8 @@ export default class Camera extends React.Component<CameraType, CameraStateType>
   }
 
   handleWebcamFailure = (error: Error) => {
-    if (permissionErrors.includes(error.name)) {
+    // $FlowFixMe
```
Comment (stefaniacardenas) [inline src/components/Camera/index.js]: What does this comment mean?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-439-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://439-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-439-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Camera/index.js (hunk 1 above)
Comment (stefaniacardenas) [reply src/components/Camera/index.js]: I've discussed this with @elhesuu directly. The PR looks good to me.

**Pull request merged**